microsoft-edge/web-platform/os-regional-settings.md

```diff
@@ -0,0 +1,85 @@
+
+* Never 
+* Limited (default) 
+* Always (not recommended). 
+
+#### The Never setting
+To minimize the risks, customers can disable the sharing of OS regional information by selecting **Never**. The **Never** option prevents websites from reading region information, but doesn't block any specific OS regional customizations that the user has made from being exposed to website content. Instead, the browser's display language is used to derive a default region. 
+
+#### The Limited setting (default)
+The default recommended value is **limited**. With this setting, the above risks are mitigated by ensuring that “incompatible” combinations of language and region aren't revealed to websites. The “limited” value ensures that only valid combinations of the user's current browser display language preference and a given region value are used. **Limited** ensures that websites won't see a regional code anomaly (though it could still look unusual per geographic inference), and also ensures that for language processing, websites will at least provide a consistent language experience (not mixing languages in weird combinations). The **limited** setting compares the language part of the regional code that is derived from the browser’s display language, and the regional part coming from the OS setting, and if the language parts match and if the resulting regional code is a valid combination, will pass the result through to the website. If the language in the message doesn't match, the feature acts as if **never** was selected (the regional information falls back to being derived from the browser’s display language). 
+
+Note: the **limited** value is similar to current Firefox behavior for exposing the regional code as described in [Firefox Source Docs documentation (mozilla.org)](https://firefox-source-docs.mozilla.org/intl/locale.html#regional-preferences), but also offers the possibility to not share this information at all. 
+
+#### The Always setting (not recommended)
+If **Always** is selected, the customer is exposed to the full set of risks noted earlier: the potential to be easily signaled out for fingerprinting, and also to potentially see website language-processing problems and inconsistencies. However, the user's OS preferences will be made available to the website without any limitations or restrictions to be sure their preferences are respected. 
+
+## Policy Controls 
+A related new policy added to Microsoft Edge provides a setting for corporate or enterprise-level default configurations: **ShareOSRegionThroughJavaScriptLocale** which accepts an integer with potential values: 
+
+|Number| Description |
+|--------|-------------------------|
+| 1 | Limited | 
+| 2 | Always | 
+| 3 | Never | 
+
+In addition, an [older policy](/DeployEdge/microsoft-edge-policies#applicationlocalevalue) (**ApplicationLocaleValue**) allows setting the Microsoft Edge locale and prevents users from changing it. 
```

Comment:
link text should match the destination heading, per style guides.  Usually/often, have a sentence w/ no links, then at end of paragraph:  See [Destination title here](url).